README.md

```diff
@@ -98,106 +35,300 @@ Class.
 **Bad:**
 
 ```html
-<div class='hello-world-wrapper'>
+<div class="hello-world-wrapper">
   <h6>Hello World!</h6>
 </div>
 ```
 
 Here I have no idea what `.hello-world-wrapper` will look like, and making
 changes means I have to search the codebase for its definition.
 
-### Only use IDs for Route Level Selectors
+---
+
+<br>
+
+## Composability with Tailwind
+
+We recently adopted the utility-first CSS framework Tailwind to help us compose and standardize our utility classes. Tailwind bundles a variety of base utility classes like _flex_, _mt-4_, _overflow-hidden_, _items-center_ and _bg-white_. We can use these utility classes to build any design directly in our markup.
+
+As developers, we spend a significant amount of time thinking of class names rather than applying them. Unfortunately, we overcomplicate this process by creating extensive naming conventions that we use seldomly or disregard completely. Tailwind remedies this process by offering us a whole suite of well-composed utility class names that work right out of the box. Check it out!
+
+<br>
+
+### Code:
 
 ```html
-<div id='homepage'>
-  <!-- Nothing on this template is outside of this ID -->
+<!-- ChatAppNotifcation.tsx -->
+
+<div
+  class="p-6 max-w-sm mx-auto bg-white rounded-xl shadow-md flex items-center space-x-4"
```

Comment:
Opinion - regardless of whether there's a matching stylesheet required for this component or not, we should always declare the component's name as the first class in the list

For example:
```
<div
  class="ChatAppNotification p-6 max-w-sm mx-auto bg-white rounded-xl shadow-md flex items-center space-x-4"
```

This helps us know which component we're expecting in the DOM

Reply:
I agree. This also makes it easier to hop to a component in React devtools, because sometimes an anonymous function component or forwardRef(component) will not have a helpful name in the component tree.